metadata/metadata-core/src/main/java/com/thinkbiganalytics/metadata/core/feed/InMemoryFeedProvider.java

```diff
@@ -368,6 +371,15 @@ public Feed findById(com.thinkbiganalytics.metadata.api.feed.Feed.ID id) {
         // TODO Auto-generated method stub
         return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)